### PR TITLE
VSTS-2710 - Fix Mobile Notification Icon

### PR DESCRIPTION
**Pull request by jessedoyle**
:elephant: :art:

* Previously we were showing the notification icon on mobile
  even if the user was not logged in. The icon was visible but
  could not be clicked.
* Create an alternate mobile partial that renders the mobile markup
  to help separate our concerns. This partial is only rendered if
  the user is logged in.
* Write decorator tests to bring coverage to 100%.
* Bump patch version.

### Broken

screen shot 2017-10-04 at 4 45 30 pm

### Fixed

screen shot 2017-10-04 at 4 45 12 pm

SEE: https://amaabca.visualstudio.com/travel_backlog/_workitems?id=2710

### PR Review Checklist
To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [X] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- ~I've added new components to the style guide (or have a PR in to add them).~
- [X] Any applicable version numbers have been updated.
- [X] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
- [X] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).
- [X] I have written a detailed PR message explaining what is being changed and why
- [X] I’ve linked to any relevant VSTS tickets
- [X] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design

### CODE REVIEW

**Comment by sineade** [comment]
:art: :art: :art:

**Comment by mvandenbeuken** [comment]
📖

File: app/views/ama_layout/_siteheader.html.erb [hunk 1]
```diff
@@ -2,18 +2,15 @@
   <div class="title-bar" data-responsive-toggle="main-menu" data-hide-for="large">
     <button class="menu-icon" type="button" data-toggle="offCanvasLeft"></button>
     <%= link_to(image_tag("ama-logo.png"), Rails.configuration.amaabca_site, class: "title-bar__logo") %>
-    <a href="#" data-toggle="offCanvasRight" data-notifications-toggle>
-      <i class="fa fa-2x fa-bell notification__icon"></i>
-      <%= navigation.notification_badge %>
-    </a>
+    <%= navigation.mobile_notification_icon %>
```
Comment (mvandenbeuken) [inline app/views/ama_layout/_siteheader.html.erb]: good call

**Comment by mvandenbeuken** [comment]
🐘 🐘 🐘

**Pull request merged**